src/FunctionApps/PITest_FunctionApp/tests/assets/encrypted.txt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN PGP MESSAGE-----
+Version: BCPG v1.55
+
+hQIMAxPUuNCDmnbzARAAnZQA97EAEuqU9d0kFM/xDnFmwLMIrHnDtozRu9kwGQhz
+cGOmnn+E8UXKvWB+9LSoO+9nWu3gIUJEo/igm0xcGICEGfN4cZe4FG9RJd8dSd+f
++tI8Q06VEd/PoTFh7XryMGpRqGFkVqgMwpKMkTjXxuEGzaPD/9J8fuEip/EgLcpD
+qQ7LWNXL97CDzrUXDewEAUgjNs5cDMR+vai9PKuHjKW8QGXJMwUSTEZ9eKbotrP/
+LuX/0Lxq/VAZCFHkUWo8K+HSo9fsm1y1UAecIr/fzwdt4JnLaVSVvWLWi2WWkO6f
+wSRQodvrFBanLmE/fNz7vwPEWRUYI+M54okBr9CYzYixVRa+22RnDIvz83qjC2nA
+YCOSQ9ikVw3YtD/eOskffDyvZ8TH6QvCPvcwxxrVJBDlsu1IoUzCHxLwOorlArsx
+V4KK2bryPUnOC6jnMSIhKEa472Ls2QnQAczubUrBy/2EkwSOmNLxjnJU7epStY6J
+Qs4rK+8+l/tGHfbohNdMbeHRME7RGfnQz+XJ+0eLBQOwVqOIvNK8OgmVCV8bgcfO
+Gm6fu7n5odoALkx0X+4BjRlTUnmKCxaDYObycCRI4enQtXv61FYrcHtTHXsEafYU
++Mw04pHKu6LeD5gMpd8IaUeTDqz5+xDKOsJWbg1lF7n5iMOF7y4gd59KUtym6o/S
+YAHUq6ZVfUTZdERma5EKHavYgGdv+1oVgJXGqhnsd4XB44Hggs4deLm8Bh0GVRIz
+eg8Ij6axIvR+u+/2wyOsxM2ULAhQOsDJKNoi1dfolLZiqmGTLAkxKhpORfwZDYmd
+jQ==
+=42tT

```

Comment:
Do we want to commit this to the repo?

Reply:
^ I think this is necessary for the test ? @bamader , obviously Nat will confirm

Reply:
Yep - this is for the test, it's a simple "TEST EICR MESSAGE" message.